docs/content/Segments.md

```diff
@@ -4,6 +4,8 @@ layout: doc_page
 Segments
 ========
 
+Druid segments contain data for a time interval, stored as separate columns. In addition, dimensions (string columns) have an inverted indexes associated with them for each dimension value.
```

Comment:
remove "an" from "have an inverted indexes".

In my reading of the existing documents, I noticed that  "indexes" and "indices" were used in different areas of the documentation.  I suspect that "indexes" is more appropriate in this context (see http://www.worldwidewords.org/qa/qa-ind2.htm for example), but whichever it is, it would be nice to be consistent.